cmd/src/users_list.go

```diff
@@ -48,16 +48,26 @@ Examples:
 		if err != nil {
 			return err
 		}
-
+		vars := map[string]interface{}{
+			"first": nullInt(*firstFlag),
+			"query": nullString(*queryFlag),
+			"tag":   nullString(*tagFlag),
+		}
+		queryTagVar := ""
+		queryTag := ""
+		if maybeTagVar, ok := vars["tag"].(*string); ok && maybeTagVar != nil {
+			queryTagVar = `$tag: String,`
+			queryTag = `tag: $tag,`
+		}
 		query := `query Users(
   $first: Int,
   $query: String,
-  $tag: String,
+` + queryTagVar + `
```

Comment:
I’m a bit confused why we must dynamically create query, as that adds complexity. Why can’t we just pass null?

Reply:
Check the issue that this fixes linked from the PR description for why